sehee/routes/postsRouter.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const postsController = require("../controllers/postsController");
+
+const router = express.Router();
+
+// 게시글 등록
+router.post("/posting", postsController.createPost);
+
+// 전체 게시글 조회
+router.get("/list", postsController.getPosts);
+
+// 유저 게시글 조회
+router.get("/list/:userId", postsController.getPostByUserId);
```

Comment:
우선 `routes/index.js` 파일을 통해 해당 엔드포인트에는 `.../posts` resource path가 이미 명시되어 있습니다.

그렇기 때문에 posting 혹은 list와 같은 추가 엔드포인를 다시 명시할 필요가 없습니다.

만약 전체 게시글을 조회한다면 이미 `.../posts` 라는 엔드포인트에서 명시된 resource 자체가 `전체 게시글 = posts`을 가리키고 있어 `list` 라는 resource를 추가적으로 적을 필요가 없습니다.
단일 게시글을 조회할 때도, 게시글을 추가할 때도 마찬가지입니다.

```suggestion
router.post("/", postsController.createPost);
router.get("/", postsController.getPosts);
router.get("/:userId", postsController.getPostByUserId);
```